README.md

```diff
@@ -54,10 +54,40 @@ There's 2 best way to reach us:
 
 Contributions are very welcome! If you have questions or issues, please let us know on [Discord](https://chat.hyperledger.org/channel/aries) or at our [bi-weekly community call](https://wiki.hyperledger.org/display/ARIES/Community+calls).
 
+## Install
+
+Install Rust: https://www.rust-lang.org/tools/install
+
+We recommend using rustup, as VCX is currently tied to Rust v1.79.x (this maintains consistency between local and CI environments). 
+
+Anoncreds and Indy require the use of openssl and zmq. These may be vendored by consuming applications, but for development installation is required. 
+
+> [!NOTE]
+> For those familiar with the Indy SDK dependencies (which is no longer in use as it has been replaced by [anoncreds-rs](https://github.com/openwallet-foundation/askar), [indy-vdr](https://github.com/hyperledger/indy-vdr/tree/main), and [aries-askar](https://github.com/openwallet-foundation/askar)) note that: 
+> - Openssl requirements are now no longer restricted to version no-longer supported 1.1 version.
```

Comment:
nice, thanks for this note!

On this line, should it say "are now no longer restricted to the no-longer supported 1.1 version."? or what is the message?

Reply:
Oops yep I butchered that sentence lol